contrib/docker/Dockerfile

```diff
@@ -22,7 +22,27 @@ RUN useradd --user-group --create-home --no-log-init --shell /bin/bash superset
 ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8
 
+# Install Dremio ODBC
+RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
 RUN apt-get update -y
+RUN apt-get install -y unixodbc-dev unixodbc-bin unixodbc
+RUN apt-get install -y alien
+RUN wget -P /home/superset/ http://download.dremio.com/odbc-driver/1.3.22.1055/dremio-odbc-1.3.22.1055-1.x86_64.rpm && \
+    alien -k -i /home/superset/dremio-odbc-1.3.22.1055-1.x86_64.rpm --scripts
+RUN apt-get update -y
+
+# Build sqlalchemy_dremio
+RUN git clone https://github.com/wschroederga/sqlalchemy_dremio.git && \
```

Comment:
This is a security issue.

More generally, we should not bloat the Superset docker image with dependencies to all databases that people may or may not need.

If people want support for Dremio (or any other db that has specific deps for that matter) and still use this image, they should do it in upper layers that they own.

I'm hoping soon the community will publish an official [lean] image, that people can use as a base to run in their environments. For now this is still just a community contributed dockerfile...

Reply:
I agree, it's probably best if you leave out the Dremio changes made to `Dockerfile` from this PR.